app/models/login.rb

```diff
@@ -0,0 +1,5 @@
+class Login < ApplicationRecord
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+end
```

Comment:
- [ ] As the last reviewer said you have to add the `confirmable` symbol to ask for email confirmation.